### ISSUE TITLE
Add support for color-based bivariate choropleth

### ISSUE THREAD

**Issue by otravers** [mapbox/mapboxgl-powerbi#339]
After some trial and error I've come to the conclusion that extrusion (as per https://github.com/mapbox/mapboxgl-powerbi/issues/188) works really well to create 3D maps at a relatively small scale with few map sections, typically the neighborhoods of a city. However things become quickly unreadable and a 3D choropleth of say, the US by state, let alone by county, is just unreadable.

I suggest adding the possibility of putting two variables in the Color field of choropleth maps, which allows the creation of legible maps like this:
![image](https://user-images.githubusercontent.com/3731532/61720167-49d34d00-ad34-11e9-83fc-e1271dac2dff.png)

Prior art:

- [Observable](https://observablehq.com/@d3/bivariate-choropleth?collection=@observablehq/maps) (the example above).
- [YouTube video](https://youtu.be/sqZ4CAdBCEo?t=345) explaining the added value of such maps.
- [Halftone with Mapbox gl](https://medium.com/@snailbones/halftone-in-mapbox-gl-70b7dd479409)

In the absence of native support, I believe this could be done manually with some DAX magic and a separate legend in a matrix using conditional formatting. I'll give it a try and post here if I get something conclusive.

**Comment by otravers** [comment]
As I suspected all it takes is 1) a few nested switch statements in DAX to determine the right color, and 2) support for expression-based coloring in the map visual, as illustrated below with the default Map visual:

![image](https://user-images.githubusercontent.com/3731532/61738243-e22ef900-ad57-11e9-87b4-e5f794a6030d.png)

TLDR: if Mapbox implements https://github.com/mapbox/mapboxgl-powerbi/issues/266 then it will be possible to create bivariate choropleth with a bit of conditional DAX (leaving aside how to render the two-axes legend).

**Comment by ryanbaumann** [comment]
cc/ @samgehret

**Comment by mustaqilmulazim** [comment]
Any updates on this issue?
